client/asset/eth/eth_test.go

```diff
@@ -555,7 +555,7 @@ func TestFundOrderReturnCoinsFundingCoins(t *testing.T) {
 	node.bal.Mul(node.bal, big.NewInt(2))
 
 	// Test funding two coins with the same id
-	coins, err = eth2.FundingCoins([]dex.Bytes{coins2[0].ID(), coins2[0].ID()})
+	_, err = eth2.FundingCoins([]dex.Bytes{coins2[0].ID(), coins2[0].ID()})
```

Comment:
in the following checkBalance, should it be referencing `coins2` instead of `coins1`